source/default/_patterns/02-molecules/tabs/tabs.css

```diff
@@ -0,0 +1,7 @@
+
+.tabs__list-link:not([aria-selected="true"]):not(:hover) {
```

Comment:
I added css since I wanted it based on the aria-selected.

Reply:
Comment both these lines of CSS with what's going on here.